test/integration/utils_test.go

```diff
@@ -294,7 +294,13 @@ func (its *integTestSuite) verifyEndpointFlow(epCfg *mastercfg.CfgEndpointState,
 	assertNoErr(err, c, "dumping flow entries")
 
 	// verify dst group entry exists
-	dstGrpFmt := fmt.Sprintf("nw_dst=%s actions=write_metadata:%d/0xfffe", epCfg.IPAddress, epCfg.EndpointGroupID)
+	var dstGrpFmt string
+	if epCfg.EndpointGroupID == 0 {
+		dstGrpFmt = fmt.Sprintf("nw_dst=%s actions=write_metadata:%d/0xfffe", epCfg.IPAddress, epCfg.EndpointGroupID)
```

Comment:
This can be just `dstGrpFmt = fmt.Sprintf("nw_dst=%s actions=write_metadata:0/0xfffe", epCfg.IPAddress)`